src/ansys/fluent/core/post_objects/post_objects_container.py

```diff
@@ -84,10 +84,29 @@ def _init_module(self, obj, mod, post_api_helper):
             if cls.__class__.__name__ in (
                 "PyLocalNamedObjectMetaAbstract",
             ) and not inspect.isabstract(cls):
+                cont = PyLocalContainer(self, cls, post_api_helper, cls.PLURAL)
```

Comment:
Overall this method contains no comments. Meanwhile, it isn't easy to understand.